recipes/coffeescript/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "coffeescript" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "4a8416fc15c0369b53dd684af186eab9956408bfdb63561ab182fb24be927a2c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jashkenas/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  
+test:
+  commands:
+    - npm list -g coffeescript
```

Comment:
Would add that it would be useful if we tested some compiling some simple CofeeScript code. Have raised as feedstock issue ( https://github.com/conda-forge/coffeescript-feedstock/issues/1 ).